content/docs/command-reference/remote/modify.md

```diff
@@ -354,6 +361,13 @@ authentication method:
   $ dvc remote modify --local myremote connection_string 'mysecret'
   ```
 
+- `account_name` - storage account name. Required if `coonection_string` is not
+  supplied. Will accompany the options below:
+
+  ```dvc
+  $ dvc remote modify myremote account_name 'myuser'
```

Comment:
not really sure about naming/style convention here:

- `myremote`/`my_remote`/`'myremote'`
- `accountname`/`account_name`/`'accountname'`
- `myuser`/`my_user`/`'myuser'`

And of course `'my_remote'`/`'account_name'`/`'my_user'`. Is there some important meaning behind this combination?

Same issue with all commands below.

Reply:
Good Qs but unrelated to this PR (specific DVC update)

But to anser, `account_name` comes from DVC. The other ones are just sample values. Currently we don't hyphenate `'myxyz'`, so this seems consistent.

Resolving (feel free to reopen if I didn't understand you).